snmp/datadog_checks/snmp/data/default_profiles/citrix-netscaler-sdx.yaml

```diff
@@ -0,0 +1,350 @@
+        tag: netscaler_sdx_xen_uuid
+  - MIB: SDX-ROOT-MIB
+    table:
+      name: netscalerTable
+      OID: 1.3.6.1.4.1.5951.6.3.2
+    symbols:
+      - name: netscaler.sdx.nsNsCPUUsage
+        OID: 1.3.6.1.4.1.5951.6.3.2.1.33
+      - name: netscaler.sdx.nsNsMemoryUsage
+        OID: 1.3.6.1.4.1.5951.6.3.2.1.35
+      - name: netscaler.sdx.nsNsTx
+        OID: 1.3.6.1.4.1.5951.6.3.2.1.36
+      - name: netscaler.sdx.nsNsRx
+        OID: 1.3.6.1.4.1.5951.6.3.2.1.37
+      - name: netscaler.sdx.nsHttpReq
+        OID: 1.3.6.1.4.1.5951.6.3.2.1.38
+    metric_tags:
+      - column:
+          name: netscaler.sdx.nsIpAddressType
+          OID: 1.3.6.1.4.1.5951.6.3.2.1.1
+        tag: netscaler_sdx_ns_ip_address_type
+        mapping:
+          0: unknown
+          1: ipv4
+          2: ipv6
+          3: ipv4z
+          6: ipv6z
+          16: dns
+      - column:
+          name: netscaler.sdx.nsIPAddress
+          OID: 1.3.6.1.4.1.5951.6.3.2.1.2
+          format: ip_address
+        tag: netscaler_sdx_ns_ip_address
+      - column:
+          name: netscaler.sdx.nsProfileName
+          OID: 1.3.6.1.4.1.5951.6.3.2.1.5
+        tag: netscaler_sdx_ns_profile_name
+      - column:
+          name: netscaler.sdx.nsName
+          OID: 1.3.6.1.4.1.5951.6.3.2.1.6
+        tag: netscaler_sdx_ns_name
+      - column:
+          name: netscaler.sdx.nsNetmaskType
+          OID: 1.3.6.1.4.1.5951.6.3.2.1.7
+        tag: netscaler_sdx_ns_netmask_type
+        mapping:
+          0: unknown
+          1: ipv4
+          2: ipv6
+          3: ipv4z
+          6: ipv6z
+          16: dns
+      - column:
+          name: netscaler.sdx.nsNetmask
+          OID: 1.3.6.1.4.1.5951.6.3.2.1.8
+          format: ip_address
+        tag: netscaler_sdx_ns_netmask
+      - column:
+          name: netscaler.sdx.nsGatewayType
+          OID: 1.3.6.1.4.1.5951.6.3.2.1.9
+        tag: netscaler_sdx_ns_gateway_type
+        mapping:
+          0: unknown
+          1: ipv4
+          2: ipv6
+          3: ipv4z
+          6: ipv6z
+          16: dns
+      - column:
+          name: netscaler.sdx.nsGateway
+          OID: 1.3.6.1.4.1.5951.6.3.2.1.10
+          format: ip_address
+        tag: netscaler_sdx_ns_gateway
+      - column:
+          name: netscaler.sdx.nsHostname
+          OID: 1.3.6.1.4.1.5951.6.3.2.1.11
+        tag: netscaler_sdx_ns_hostname
+      - column:
+          name: netscaler.sdx.nsDescription
+          OID: 1.3.6.1.4.1.5951.6.3.2.1.12
+        tag: netscaler_sdx_ns_description
+      - column:
+          name: netscaler.sdx.nsVersion
+          OID: 1.3.6.1.4.1.5951.6.3.2.1.13
+        tag: netscaler_sdx_ns_version
+      - column:
+          name: netscaler.sdx.nsVmDescription
+          OID: 1.3.6.1.4.1.5951.6.3.2.1.30
+        tag: netscaler_sdx_ns_vm_description
+      - column:
+          name: netscaler.sdx.nsThroughput
+          OID: 1.3.6.1.4.1.5951.6.3.2.1.31
+        tag: netscaler_sdx_ns_throughput
```

Comment:
Do  `nsThroughput` have high cardinality? I'm wondering if it's an configuration value or it might vary frequently.

```

nsThroughput OBJECT-TYPE 
	SYNTAX  OCTET STRING
	MAX-ACCESS  read-only
	STATUS  current
	DESCRIPTION
		"Assign throughput in Mbps to VM Instance"
	::= { netscalerEntry 31 }

```